Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -60,6 +77,15 @@ - (void)viewDidLoad
         [sself.bottomSheetVC setDrawerPositionWithPosition:[PulleyPosition partiallyRevealed] animated:YES completion:nil];
         sself.initialDataIsLoaded = YES;
     }];
+    [[NSNotificationCenter defaultCenter] addObserverForName:@"ARLocalDiscoveryCityGotScrollView" object:nil queue:nil usingBlock:^(NSNotification * _Nonnull note) {
+            if (!self.rnScrollView) {
```

Comment:
Shouldn't this be a weak ref to `self`?

(Also, looks like the `sself` var above should be `wself`.)

Reply:
Yup to both 👍